Components/Queues/QueueManager.php

```diff
@@ -147,10 +147,15 @@ public function send(string $queueName, string $message, array $headers = [])
         return $this->exchange->publish($message, $queueName, AMQP_NOPARAM, $headers);
     }
 
-    public function addConnection(\AMQPConnection $connection)
+    public function addConnection($connection)
     {
         $this->connections[] = $connection;
 
         return $this;
     }
+    
+    public function getConnections()
```

Comment:
Is this new method used somewhere? I am not able to find any reference on the merge request

Reply:
Ok. I removed the function